src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -333,23 +333,33 @@ private void LookupForMatchingGitHubRepository(BuildData mergedBuild)
 
             using (var client = new HttpClient())
             {
-                string[] segments = mergedBuild.AzureDevOpsRepository.Split('/');
-                string repoName = segments[segments.Length - 1];
-                int index = repoName.IndexOf('-');
+                Uri repoAddr = new Uri(mergedBuild.AzureDevOpsRepository);
+                string repoIdentity = string.Empty;
 
-                StringBuilder builder = new StringBuilder(repoName);
-                builder[index] = '/';
+                if (repoAddr.Host.Equals("github.com", StringComparison.OrdinalIgnoreCase))
```

Comment:
>"github.com" [](start = 41, length = 12)

Nit: define a variable for this and reuse it in line 355

Reply:
Didn't get you comment :-/ - can you please clarify?